kadi/config.py

```diff
@@ -79,6 +79,14 @@ class Conf(ConfigNamespace):
         "of command generation prior to that patch.",
     )
 
+    maneuver_nman_step_time = ConfigItem(
+        300.0, "Time step (sec) for NMAN maneuver generation in kadi states."
```

Comment:
Maybe just a two-liner here in the config item?  "Time step (sec) for NMAN maneuver generation in kadi states.  Applied as the step_max kwarg of CxoTime.linspace so effective step size may be smaller."